src/refdb_fs.c

```diff
@@ -254,7 +254,9 @@ static int loose_lookup_to_packfile(refdb_fs_backend *backend, const char *name)
 	if ((error = loose_parse_oid(&oid, name, &ref_file)) < 0)
 		goto done;
 
-	git_sortedcache_wlock(backend->refcache);
+	if ((error = git_sortedcache_wlock(backend->refcache)) < 0) {
+		goto done;
+	}
```

Comment:
```suggestion
	if ((error = git_sortedcache_wlock(backend->refcache)) < 0)
		goto done;
```

we can lose the braces for consistency with the other blocks above. same in L763 for consistency with this.

Reply:
Thanks, dunno how I missed that